test/cpp/microbenchmarks/bm_callback_streaming_ping_pong.cc

```diff
@@ -0,0 +1,155 @@
+#include "test/cpp/microbenchmarks/callback_streaming_ping_pong.h"
```

Comment:
All the new files need copyright notices; you can copy them from the existing files but with the year updated to 2019